packages/cli/src/lib/history/history.model.ts

```diff
@@ -0,0 +1,7 @@
+import { LogOptions } from 'simple-git';
+import { HistoryOnlyOptions } from '@code-pushup/core';
+
+export type HistoryCliOptions = {
+  targetBranch?: string;
+} & Pick<LogOptions, 'maxCount' | 'from' | 'to'> &
+  Required<HistoryOnlyOptions>;
```

Comment:
Why is this `Required`? The `forceCleanStatus` does not have a default value.

Reply:
removed